iosApp/PhongKMMIC/Sources/Supports/Extensions/KMM/ViewModels/SplashViewModel+ObservableObject.swift

```diff
@@ -0,0 +1,11 @@
+//
+//  SplashViewModel+ObservableObject.swift
+//  PhongKMMIC
+//
+//  Created by Phong Vo on 20/04/2023.
+//  Copyright © 2023 Nimble. All rights reserved.
+//
+
+import shared
+
+extension SplashViewModel: ObservableObject {}
```

Comment:
I don't think putting the ViewModel into the Shared module is a good idea, what's happened if we need to use Combine in this screen 🤔

Reply:
Maybe a wrapper like Bliss did. Or we can transform Kotlin Flow to Combine, there's many resources about that, and I want to validate them.